zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -2073,7 +2073,7 @@ public Object call() throws Exception {
       record(path, null, startT, ZkClientMonitor.AccessType.WRITE);
     } catch (Exception e) {
       recordFailure(path, ZkClientMonitor.AccessType.WRITE);
-      LOG.warn("zkclient {}, Failed to delete path {}! ", _uid, path, e);
+      LOG.debug("zkclient {}, Failed to delete path {}! ", _uid, path, e);
```

Comment:
Umm is this correct? There are 3 parameters for 2 positional brackets.

Reply:
I think the first two parameters are for the positional brackets. The last one since it's an exception it is thrown as a separate message. Let me know if I'm mistaken